Formula/zeromq.rb

```diff
@@ -20,35 +20,22 @@ class Zeromq < Formula
     depends_on "libtool" => :build
   end
 
-  option "with-libpgm", "Build with PGM extension"
-  option "with-norm", "Build with NORM extension"
-  option "with-drafts", "Build and install draft classes and methods"
```

Comment:
You've removed the --with-drafts option which means we won't be able to build with the draft api enabled. Could we keep this option or add --enable-drafts to the configure step by default?

Reply:
Our analytics show that 44 people out of ~44.000 used it in the past year. I'd say the api isn't really needed.

Reply:
That's a shame. I'm one of those 44 people so I guess I won't be able to use Homebrew to obtain ZeroMQ.

Reply:
You still can use homebrew, but suggested path is to make your own tap. Here are the docs on how to do it https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap

For core brew repo it's hard to test all the possible combination of flags on CI (and we don't test them) as a consequence often flags broken after version upgrades and consequentially for the people like you we bring broken upgrade. While if it critical software for you, you know when to upgrade and when flags work.

As @fxcoudert mentioned before, there is more info in #31510 with the reasoning behind this.

Reply:
Thanks @igas. I'll create a tap preserving the draft flags until ZeroMQ gets around to marking the features I use as stable.